src/libPMacc/include/cuSTL/cursor/navigator/PlusNavigator.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2015 Heiko Burau
+ *
+ * This file is part of libPMacc.
+ *
+ * libPMacc is free software: you can redistribute it and/or modify
+ * it under the terms of either the GNU General Public License or
+ * the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * libPMacc is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with libPMacc.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+namespace PMacc
+{
+namespace cursor
+{
+
+/** Navigator which simply adds a jump of any type to a marker
```

Comment:
IMO: this means a random access

I prefer `RandomAccessNavigator` because that is what you get by add a jump to the marker.

@ax3l @Flamefire What you think? 

I wait a while for an answer else I merge it.